gh-108277: Add wrapper for timerfd_create, timerfd_settime, and timerfd_gettime to os module (#108382)
python/cpython: Doc/whatsnew/3.13.rst

  documented or exposed through ``dis``, and were not intended to be
  used externally.

os
--

> * Add :func:`os.timerfd_create`, :func:`os.timerfd_settime`, :func:`os.timerfd_gettime`, :func:`os.timerfd_settime_ns`,
>   and :func:`os.timerfd_gettime_ns` to provide a low level interface for Linux's timer notification file descriptor.
>   (Contributed by Masaru Tsuchiyama in :gh:`108277`.)

Reviewer: Put the main object first (the interface to the timerfd system)
```suggestion
* Add a low level interface for Linux's timer notification file descriptors
  via :func:`os.timerfd_create`,
  :func:`os.timerfd_settime`, :func:`os.timerfd_settime_ns`,
  :func:`os.timerfd_gettime`, and :func:`os.timerfd_gettime_ns`.
  (Contributed by Masaru Tsuchiyama in :gh:`108277`.)
```

* Add :const:`os.TFD_NONBLOCK`, :const:`os.TFD_CLOEXEC`, :const:`os.TFD_TIMER_ABSTIME`, and :const:`os.TFD_TIMER_CANCEL_ON_SET`

>   (Contributed by Masaru Tsuchiyama in :gh:`108277`.)

Reviewer: Are these distinct from the timerfd interface?

Author: @AA-Turner 
This is a part of the timerfd interface.
Should I remove?

Author: @AA-Turner 
Merged the sections.
